sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_user_agent.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import platform
+from ._version import VERSION
+
+USER_AGENT = "azsdk-python-azure-ai-formrecognizer/{} Python/{} ({})".format(
```

Comment:
azure-core should be responsible for Python and platform, look in other SDK for "sdk_moniker" kwarg

Reply:
Thank you for pointing this out! Removed my user-agent file, using sdk-moniker now.